src/data/multiset/powerset.lean

```diff
@@ -236,4 +236,21 @@ theorem powerset_len_mono (n : ℕ) {s t : multiset α} (h : s ≤ t) :
 le_induction_on h $ λ l₁ l₂ h, by simp [powerset_len_coe]; exact
   ((sublists_len_sublist_of_sublist _ h).map _).subperm
 
+@[simp] theorem powerset_len_empty {α : Type*} (n : ℕ) {s : multiset α} (h : s.card < n) :
```

Comment:
Should this be in the default simp-set?

Reply:
I don't know, it seemed fine to me, the side condition seems to be in a natural enough form that it might actually fire for someone.

Reply:
Any further thoughts on this?

Reply:
I think these are okay simp lemmas. I don't love adding conditional lemmas to the simp set but these are at least keyed to `powerset_len` which isn't something that shows up all that often.